submit_and_compare/submit_and_compare.py

```diff
@@ -196,6 +196,10 @@ class SubmitAndCompareXBlock(XBlock):
     """
     Main functions
     """
+    # Decorate the view in order to support multiple devices e.g. mobile
+    # See: https://openedx.atlassian.net/wiki/display/MA/Course+Blocks+API
+    # section 'View @supports(multi_device) decorator'
+    @XBlock.supports("multi_device")
```

Comment:
This looks great! 

One little nitpick: By convention, our team tries to use single-quotes for non-interpolated strings, wherever possible.

so `@XBlock.supports('multi_device')`